### PR TITLE
Update project documentation

### PR DESCRIPTION
**Pull request by jonseitz**
This builds on the previous version of the `README` by providing more details about the individual components of the app, providing links to relevant external resources (including the `sec-directory-server` project) and making some recommendations about separating development and test indices.

I've also added our standard `CODE_OF_CONDUCT` document, as well as a copy of the BSD 3-clause license that we've used in the past. 

Most importantly, there are badges now.

Please let me know if you have any suggestions for other details that we should add to the README, of if there's anything here that's not totally clear. I know, the production section is definitely a little thin, but I expect we can add to that once we actually have a deployment out in the wild.

### CODE REVIEW

**Review by natalynyu — CHANGES_REQUESTED** [review]
This is looking good. I just had a few comments on the README portion.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
-## Commands
+## Algolia
+
+The backend of the application is the Algolia search index defined in the `REACT_APP_ALGOLIA_INDEX`/`REACT_APP_ALGOLIA_APP_ID` environment variable. You'll also need to provide an API key in `REACT_APP_ALGOLIA_API_KEY`, which should only have "search" permissions for the specified index. **This API key will be included verbatim in the client code, so you should never use an admin key, or any key with write permission.** Best practice is [to create a "Secured API Key"][api-key] scoped only to the index you want to use.
+
+## Local Development
+
+`npm run start` will bring up a live development environment with hot-reloading. Once it's started, you can view the app in the browser at http://localhost:3000/.
+
+As we're building for specific 55" touch screen displays ([Planar PS5561T][planar]) in portrait mode, the design of the page is fixed to a 1080x1920 rectangle with scrolling disabled. To view the application on a standard screen, you can use the [Responsive Design Mode][rdm] built into most browsers to emulate a 1080x1920 viewport.
```
[the remark this answers is not in the document]
Comment (jonseitz) [reply README.md]: Yeah, probably doesn't hurt to mention it. Added that in 376d2bb.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+
+The only outbound data connections are to Algolia, so the server project does not need to be running at the same time.
+
+### Development Index
+
+Ideally, each developer working on the project should create their own Algolia index (and corresponding API Key) to use while developing. You can use the same index for both client and server operations, though the server application does require an API key with write permission.
```
[the remark this answers is not in the document]
Comment (jonseitz) [reply README.md]: Yep, probably good to repeat that often. Added in 64ce2b0

**Review by chrxr — APPROVED** [review]
Looks good! Thanks.

**Pull request merged**